frontend/src/components/eMIB/ActionViewTask.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "../../css/collapsing-item.css";
+import LOCALIZE from "../../text_resources";
+import { actionShape } from "./constants";
+
+const styles = {
+  responseTypeIcon: {
+    color: "white",
+    margin: "0 8px",
+    padding: 3,
+    backgroundColor: "#00565E",
+    border: "3px solid #009FAE",
+    borderRadius: 4
+  },
+  responseType: {
+    color: "#00565E",
+    textDecoration: "underline"
+  },
+  hr: {
+    margin: "16px 0 16px 0"
+  },
+  editButton: {
+    float: "right"
+  }
+};
+
+class ActionViewTask extends Component {
+  static propTypes = {
+    action: actionShape
+  };
+
+  render() {
+    const { action } = this.props;
+    return (
+      <div>
+        <div
+          aria-label={LOCALIZE.ariaLabel.taskDetails}
+          tabIndex="0"
+          aria-describedby="task-content"
+        >
+          <div id="task-content" role="dialog">
```

Comment:
These individual pieces are not dialogs. We need to be managing focus other ways, because this is going to cause us problems. Here's example's using tab index: https://developers.google.com/web/fundamentals/accessibility/focus/using-tabindex

Reply:
I simplified the aria-labels and roles structure.
See commit #https://github.com/code-for-canada/project-thundercat/pull/186/commits/a4a6955f4ce3899eeb653e1edf3bb901a86901a8

Is that what you had in mind?